README.md

```diff
@@ -157,14 +156,14 @@ the Serilog `LogEvent` and .NET `Activity` context via the `IncludedData` flags
 Log.Logger = new LoggerConfiguration()
     .WriteTo.OpenTelemetry(options =>
     {
-        options.Endpoint = "http://127.0.0.1:4317/v1/logs";
+        options.Endpoint = "http://127.0.0.1:4317";
         options.IncludedData: IncludedData.MessageTemplate |
                               IncludedData.TraceId | IncludedData.SpanId;
     })
     .CreateLogger();
 ```
 
-The example shows the default value; `IncludedData.MessageTemplateMD5HashAttribute` can
+The example shows the default value; `IncludedData.MessageTemplateMD5HashAttribute`~~~~ can
```

Comment:
Should the '~~~~' be there?

Reply:
Oops! Sorry about that, fixed now :-)

(`<Ctrl + S>` saves all in Rider, except if the Markdown editor is active, in which case it adds strikethrough 🤦‍♂️ )

Reply:
You do know we have autosave these days, dad?
(One of the very few annoying things about Rider)

Reply:
The terrible thing is - I do! But the muscle memory refuses to die 😆